boxoffice/static/js/views/sidebar.js

```diff
@@ -0,0 +1,47 @@
+import {SideBarModel} from '../models/sidebar.js';
+import {SideBarTemplate} from '../templates/sidebar.html.js';
+
+export const SideBarView = {
+  init: function(sidebar_item, org, ic) {
+    this.on = true;
+
+    this.ractive = new Ractive({
+      el: '#sidebar',
+      template: SideBarTemplate,
+      data: {
+        sidebarMobileOn: false,
+        sideBar: SideBarModel.getItems(org, ic),
+        activeItem: sidebar_item,
```

Comment:
I recommend accepting an explicit boolean value, instead of comparing this with the `title`. This way it doesn't become too tightly coupled to the title.